add RocketPenguin

gamekiller2048 opened the pull request.

acornitum commented:
I really like this game! The gameplay is really smooth, the mechanics are super intuitive. I also really like the art - the walking animations are amazing! I also really like how the penguins break and crumble to the ground when hit - that's a super nice detail. Kudos to you! 

This game is close to getting merged, but I have three suggestions before I do so!
- make the penguins from the right come from beyond the edge of the screen, instead of just the edge of the screen! They lowkey look like they're teleporting in right now.
- After the player dies, don't just close the game! Make a death screen and give the player the choice to restart.
- Something where the game gets harder as time goes on would be nice - from my experiences of playing the game, it goes pretty infinite, and I would really like there to be something where the enemies either get faster or take more health or something to increase the difficulty.

I highly enjoyed this game. I can't wait to play it again when you've implemented those! 

_Note: don't open a new PR, just commit changes to this current thread + send a comment when changes have been made!_

gamekiller2048 commented:
Thanks for the feedback! 
Yeah the game was missing things as I made this in a day forgetting about the deadline.

Changes:
- restart screen
- enemies speed up and I added another enemy (just a bigger version of the current one lmao)
- fixed the offscreen bug
- and some other stuff

lmk if you have any other suggestions or bufixes you think I should do before merging.

acornitum commented:
This is great! I really enjoyed playing this game - I also really like how the game over screen slides down really smoothly! It's really fun, and once again, the art + animations are amazing.

I'm going to merge this! Fill out the form here: https://forms.hackclub.com/t/isDdJtrnrvus

Thanks so much for submitting!
